how-to/workspace-platform-starter/client/src/framework/shapes/splash-shapes.ts

```diff
@@ -46,4 +46,9 @@ export interface SplashScreenProviderOptions {
 	 * The border color, defaults to theme border.
 	 */
 	borderColor?: string;
+
+	/**
+	 * The taskbar icon. Falls back to icon if not provided.
+	 */
+	taskbarIcon?: string;
```

Comment:
Hi Cameron,

Few quick things:

1) Can you run the following commands from wps: npm run generate-types, then npm run generate-schema? 
2) Can you update the changelog.md to put this as an enhancement
3) Then from the root run npm run validate just to make sure the listing rules are all applied.